src/gamma.jl

```diff
@@ -317,6 +317,15 @@ function zeta(s::ComplexOrReal{Float64}, z::ComplexOrReal{Float64})
     return ζ
 end
 
+
+"""
+    hurwitz_zeta(s, z)
+
+    An alias for zeta(s, z)
+"""
+hurwitz_zeta(s::ComplexOrReal{Float64}, z::ComplexOrReal{Float64}) = zeta(s, z)
```

Comment:
Yeah, aliases like this aren't typically provided. If it isn't obvious what something is, that means the documentation should be better. 🙂

Typical Julia conventions prefer more descriptive names for functions, and function names are typically all lowercase. Since `polylog` is the more descriptive name (and jives with `polygamma`), I think we should just stick to that one.